### ISSUE TITLE
Compiler pass incompatibility with upcoming pegjs 0.11

### ISSUE THREAD

**Issue by lindner** [metadevpro/ts-pegjs#34]
It appears that pegjs changed their API for compiler passes

```
  // pegjs 0.10  api pass(ast, options)
  // pegjs 0.11+ api pass(ast, config, options);
```

It should be fairly easy to special case that.

**Comment by pjmolina** [comment]
Thanks. Good point. 
However v. 0.11 is not yet published as stable in npmjs, but as `dev` release.
Therefore I will stick to 0.10 as stable version and incorporate your change when 0.11 is marked as stable.
Works for you @lindner in the meantime?

**Comment by lindner** [comment]
This is fine; but the change is compatible with 0.10 and 0.11 so it is safe
to add now.

I sure wish they would release a new pegjs, there's a lot of useful work
that hasn't been released yet..

**Comment by pjmolina** [comment]
Oh I see. Then, agreed to merge it right now.

**Comment by pjmolina** [comment]
Merged and published as version: `ts-pegjs@0.2.4`
